### DISCUSSION TITLE
New mojojs plugin for running in AWS Lambda

### DISCUSSION THREAD

**Discussion by pplu** [mojolicious/mojo.js#71] [General]
Hello,

Just wanted to announce that I've published a new module to run mojojs applications inside AWS Lambda.
    https://www.npmjs.com/package/mojo-plugin-lambda
 It's rough and very basic, but it works for me and a very small application that I've written. Hope it helps others embrace mojojs.

Thanks for mojojs and the brilliant documentation, clear code, examples, and complete documentation (which has been a big help for getting this working (I haven't written JS for over a decade).

**Comment by kraih** [comment]
Cool, think this could be quite a bit more efficient if it used the same APIs as the [CGI](https://github.com/mojolicious/mojo.js/blob/main/src/cgi.ts) binding. And i wonder if we should add a mechanism for disabling the command line interface from a hook, maybe like:
```
app.addAppHook('command:before', async app => {
  if (process.env.AWS_LAMBDA_FUNCTION_NAME !== undefined) return false;
});
```

**Reply by kraih** [reply]
Will be in the next release: https://github.com/mojolicious/mojo.js/commit/a548c6fbcf3b795b2a30c842f82307bbc0b55f88

**Reply by kraih** [reply]
Released with 1.2.0.
